oligo_designer_toolsuite/oligo_pre_filter/_filter_base.py

```diff
@@ -14,6 +14,7 @@ def __init__(self) -> None:
     def apply(self, sequence):
         """Applies teh filters and returns if the sequence fulfillts the constraints and the additional features computed in a dictionary.
```

Comment:
Typos:
... reimplemented in **the** ...
The **additional** computed features...